apps/prairielearn/elements/pl-external-grader-results/pl-external-grader-results.py

```diff
@@ -1,8 +1,11 @@
+from typing import List, Optional
```

Comment:
Can we use the lowercase `list` (and `tuple`) instead throughout this PR? We're on Python >=3.9, so the lowercase annotations should work and reduces the number of imports.

Reply:
Ah yup, can do. TIL those lowercase type annotations were the new way of doing things (I thought they were the old style).